config/locales/doorkeeper_openid_connect.en.yml

```diff
@@ -0,0 +1,23 @@
+en:
```

Comment:
You can just add extra local files like this - if the gem doesn't make these available automatically then you'll have to add them to the main `en.yml` file.

Reply:
I was confused with this file - it was generated by gem's installation.

Will move them to en.yml,